scripts/openmc-track-to-vtk

```diff
@@ -41,9 +42,8 @@ def main():
 
     # Check input file extensions.
     for fname in args.input:
-        if not (fname.endswith('.h5') or fname.endswith('.binary')):
-            raise ValueError("Input file names must either end with '.h5' or"
-                             "'.binary'.")
+        if not fname.endswith('.h5'):
+            raise ValueError("Input file names must an HDF5 file.")
```

Comment:
typo: "must an HD5F file" -> "must be an HDF5 file." I've seen some HDF5 files ending with ".hdf5" before, so might consider including that as an accepted option. And personally I think a message like "Input HDF5 files must have an ".h5" or ".hdf5"  extension" is more explicit and transparent than the current message.